src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -304,6 +301,26 @@ impl<'tcx> RegionInferenceContext<'tcx> {
         self.definitions.indices()
     }
 
+    /// Iterates through each row and the accompanying bit set.
+    pub fn liveness_constraints<'a>(
+        &'a self
+    ) -> impl Iterator<Item = (RegionVid, &'a SparseBitSet<RegionElementIndex>)> + 'a {
+        self.liveness_constraints.iter_enumerated()
+    }
+
+    /// Number of liveness constaints in region inference context.
```

Comment:
Typo: s/containts/constraints

Reply:
I would usually call this `num_liveness_constraints`, but then again I've been trying to use fewer and fewer contractions...

Reply:
Fixed this.